README.md

```diff
@@ -24,6 +22,17 @@ pytest
 lit tests/filecheck
 ```
 
+## Generating executables through MLIR
+
+xDSL can generate executables using MLIR as the backend. To use this functionality, make sure to install the [MLIR Python Bindings](https://mlir.llvm.org/docs/Bindings/Python/). Given an input file `input.xdsl`, that contains IR with only the mirrored dialects found in `src/xdsl/dialects` (arith, memref, func, cf, scf, and builtin), run:
+
+```
+mlir-opt --convert-scf-to-cf --convert-cf-to-llvm --convert-func-to-llvm --convert-arith-to-llvm --convert-memref-to-llvm --reconcile-unrealized-casts *input.xdsl* | mlir-translate --mlir-to-llvmir > tmp.ll
```

Comment:
How can we directly use an `input.xdsl` file in MLIR?
Don't we need to use mlir_converter or something like that?

Reply:
Yes, you are right, it should actually be passed through the mlir_converter aswell. I will think about how to properly integrate the tooling from the ChocoPy repo into xdsl. (I will probably add `.mlir` as a target to xdsl_opt, similar to how we did in choco-opt)